ci-operator/jobs/openshift/router/openshift-router-master-presubmits.yaml

```diff
@@ -130,31 +129,95 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+    trigger: (?m)^/test( | .* )e2e-metal-ipi-ovn-dualstack,?($|\s.*)
   - agent: kubernetes
-    always_run: false
+    always_run: true
```

Comment:
Looks like the optional settings on the router metal IPI jobs got clobbered. @stbenjam ill make a follow up PR but just raising awareness in case other repos are affected similarly.